### PR TITLE
stringtie 2.1.3b

### PR DESCRIPTION
**Pull request by heavywatal**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/stringtie.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Stringtie < Formula
   # cite Pertea_2015: "https://doi.org/10.1038/nbt.3122"
   desc "Transcript assembly and quantification for RNA-Seq"
-  homepage "https://ccb.jhu.edu/software/stringtie"
-  url "https://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.6.tar.gz"
-  sha256 "9dbdf01ab3282dc6eb647409920b53f8e0df45f4c45faf9d6b5ca135a3738ee8"
+  homepage "https://github.com/gpertea/stringtie"
```
Comment (sjackman) [inline Formula/stringtie.rb:4]: Please revert this line. The official web site has not changed. See https://github.com/gpertea/stringtie#readme

Comment (heavywatal) [reply Formula/stringtie.rb]: https://ccb.jhu.edu/software/stringtie redirects to insecure http, and I thought it was a cause of this error:
https://github.com/brewsci/homebrew-bio/runs/798027175#step:4:62
but now it seems not to be the case.

File: Formula/stringtie.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Stringtie < Formula
   # cite Pertea_2015: "https://doi.org/10.1038/nbt.3122"
   desc "Transcript assembly and quantification for RNA-Seq"
-  homepage "https://github.com/gpertea/stringtie"
+  homepage "https://ccb.jhu.edu/software/stringtie"
```
Comment (sjackman) [inline Formula/stringtie.rb:4]:
```suggestion
  homepage "https://ccb.jhu.edu/software/stringtie/"
```

Comment (sjackman) [reply Formula/stringtie.rb:4]: https://ccb.jhu.edu/software/stringtie redirects to the insecure URL http://ccb.jhu.edu/software/stringtie/
```console
$ curl -is https://ccb.jhu.edu/software/stringtie | grep Location
Location: http://ccb.jhu.edu/software/stringtie/
```
It works with the trailing slash: https://ccb.jhu.edu/software/stringtie/

File: Formula/stringtie.rb (hunk 1 above)
Comment (sjackman) [reply Formula/stringtie.rb]: https://ccb.jhu.edu/software/stringtie/ with the trailing slash works!

**Comment by sjackman** [comment]
Merged. Thanks again for your contributions to Brewsci/bio, Watal!

**Pull request merged**